test-clients/yahcli/config.yml

```diff
@@ -3,7 +3,7 @@ networks:
   localhost:
     allowedReceiverAccountIds: []
     nodes:
-      - { id: 0, account: 3, ipv4Addr: host.docker.internal }
+      - { id: 0, account: 3, ipv4Addr: 127.0.0.1}
```

Comment:
I'm confused which this is supposed to be. I think I've seen it change a couple of times??

Reply:
need to be reverted by mistake slicked in